ArduPlane/GCS_Mavlink.cpp

```diff
@@ -442,13 +445,6 @@ bool GCS_MAVLINK_Plane::try_send_message(enum ap_message id)
         plane.send_wind(chan);
         break;
 
-    case MSG_PID_TUNING:
-        if (plane.control_mode != MANUAL) {
```

Comment:
I think you have lost this check for manual mode.

Reply:
although its not like rover has it, I think it doesn't matter either way

Reply:
I've re-added the check.  I'm not sure it's actually a *good* check in that we won't be using some of these PIDs depending on your mode, so excluding MANUAL (just because it should be using none) seems a bit off.

Still, trying not to make too many behaviour changes.

Thanks!